balloon.py

```diff
@@ -0,0 +1,39 @@
+from pygments.lexer import *
+from pygments.token import *
+
+class CustomLexer(RegexLexer):
+    name = 'Balloon'
+    aliases = ['balloon']
```

Comment:
Please check if aliases should also have `bl`. If `js` is in the alises for javascript, then I think you should add `bl` too.